esmvaltool/cmorizers/data/formatters/datasets/merra2.py

```diff
@@ -100,11 +147,50 @@ def _extract_variable(in_files, var, cfg, out_dir):
                 var['short_name'], ', '.join(in_files))
     attributes = deepcopy(cfg['attributes'])
     attributes['mip'] = var['mip']
+    attributes['raw'] = var['raw']
+    pairwise_ops = ["+", "-", ":"]
+    for oper in pairwise_ops:
+        if oper in var['raw']:
+            components = var['raw'].split(oper)
+            if len(components) == 2:
+                attributes['component_raw_1'] = components[0]
+                attributes['component_raw_2'] = components[1]
+                attributes['component_operation'] = oper
+                break
     cmor_table = CMOR_TABLES[attributes['project_id']]
     definition = cmor_table.get_variable(var['mip'], var['short_name'])
 
     cube = _load_cube(in_files, var)
 
+    # keep the following raw cube attributes
+    attrs_to_keep = [
+        "institution", "Institution",
+        "institute_id", "VersionID",
+        "experiment_id",
+        "source", "Source",  # overrides empty string default
+        "model_id", "ModelID",
+        "contact", "Contact",
+        "references",
+        "tracking_id",
+        "mip_specs",  # described by "mip" already
+        "source_id", "SourceID",
+        "product", "Product",
+        "frequency", "Frequency",
+        "creation_date",
+        "project_id", "ProjectID",
+        "table_id", "TableID",
+        "title", "Title",
+        "modeling_realm",
+        "doi",
+        "VersionID",  # described by "version" already
+    ]
```

Comment:
@glpotter I have added this list of global attributes for the cmorizer to add whatever global attribute finds in the original (raw) data and is part of this list, and put it in the cmorized data. Note that since the attributes vary from file/variable to file/variable, the list is not exhaustive, so please add any other attribute you think you will need in the cmorized data :+1: This is in reference to your https://github.com/ESMValGroup/ESMValTool/pull/2774#issuecomment-1243038556